src/containers/datasets/contextual-layers/basemaps-planet/analytic/types.d.ts

```diff
@@ -0,0 +1,4 @@
+export type UseQueryParams = {
+  name_is?: string; // returns up to one result that exactly matches the provided value.
+  name_contains?: string; // returns only results that contain the fragment, case-insensitive.
+};
```

Comment:
What's this file? Doesn't look like you are using it.

Reply:
I was planning to get all series dynamically, but as I am not sure yet what base maps we are going to use I mocked them